apps/web/pages/api/social/og/image.tsx

```diff
@@ -75,12 +74,10 @@ export default async function handler(req: NextApiRequest) {
         imageType,
       });
 
-      const title_ = md.render(title).replace(/(<([^>]+)>)/gi, "");
```

Comment:
we don't need this as we are stripping the markdown content in the user's public profile page